target/linux/lantiq/image/Makefile

```diff
@@ -560,42 +593,49 @@ TARGET_DEVICES += zyxel_p-2812hnu-f1
 
 define Device/zyxel_p-2812hnu-f3
   $(Device/NAND)
+  DEVICE_VENDOR := ZyXEL
+  DEVICE_MODEL := P-2812HNU-F3
   BOARD_NAME := P2812HNUF3
   DEVICE_DTS := P2812HNUF3
-  DEVICE_TITLE := ZyXEL P-2812HNU-F3
   DEVICE_PACKAGES := kmod-rt2800-pci wpad-basic kmod-usb-dwc2
   SUPPORTED_DEVICES += P2812HNUF3
 endef
 TARGET_DEVICES += zyxel_p-2812hnu-f3
 
 define Device/arcadyan_arv7519rw22
+  DEVICE_VENDOR := Livebox
+  DEVICE_MODEL := Astoria ARV7519RW22
   IMAGE_SIZE := 31232k
   DEVICE_DTS := ARV7519RW22
-  DEVICE_TITLE := Livebox Astoria ARV7519RW22
   DEVICE_PACKAGES := kmod-usb-dwc2
   SUPPORTED_DEVICES += ARV7519RW22
 endef
 TARGET_DEVICES += arcadyan_arv7519rw22
 
 define Device/alphanetworks_asl56026
+  DEVICE_VENDOR := BT Openreach
+  DEVICE_MODEL := ECI
+  DEVICE_VARIANT := VDSL Modem V-2FUb/I
```

Comment:
Another one I missed: This is not the variant but belongs to the title.

Reply:
Done